package.json

```diff
@@ -3,11 +3,14 @@
     "version": "0.1.0",
     "description": "Fluid is an open, collaborative project to improve the user experience and inclusiveness of open source software.",
     "scripts": {
-        "start": "npm run clean && npm run serve",
-        "serve": "eleventy --serve",
-        "build": "rimraf dist && eleventy",
-        "lint": "grunt lint",
-        "clean": "rimraf dist"
+        "start": "cross-env NODE_ENV=development run-p -l start:*",
+        "start:webpack": "webpack --config webpack.dev.js ",
+        "start:eleventy": "eleventy --serve",
```

Comment:
I think you're missing a clean function for the `npm start` flow. At least it seemed that when I ran it, it was picking up old values.

Reply:
Resolved in https://github.com/fluid-project/fluidic-11ty/pull/31/commits/8eebc5f514889d9aaaef54153fdc4ec7a623f769.